### DISCUSSION TITLE
Fields for Fiscal/Environmental/Employment Legislation

### DISCUSSION THREAD

**Discussion by ghost** [OP-TED/eForms-SDK#526] [Q&A] [answered]
I am looking for a description of the fields for Fiscal/Environmental/Employment Legislation (eg GR-Lot-FiscalLegis)
I'm wondering about the IDs in this groups and how to explain to our users what to fill in. The labels are not really meaningful.
I could not find anything about this fields in the regulation nor in the directive. Are there any explaning texts in the SDK?

**Comment by YvesJo** [comment, marked as answer]
Hi,
I guess you're referring to the OPT-111-xxx fields.
These fields are required to satisfy the schema. There is no control in place regarding the content. It may be used to insert the legislation Identifier (e.g. ELI number) or, when none is known, a keyword like illustrated at https://docs.ted.europa.eu/eforms/latest/schema/parties.html#_legislation_information_provider
Also for non-technical questions, please get in touch with the helpdesk instead.
Thanks
